src/sql/workbench/services/jobManagement/browser/jobManagementUtilities.ts

```diff
@@ -40,16 +40,28 @@ export class JobManagementUtilities {
 		return bool ? nls.localize('agentUtilities.yes', "Yes") : nls.localize('agentUtilities.no', "No");
 	}
 
-	public static convertToNextRun(date: string) {
-		return date.includes('1/1/0001') ? nls.localize('agentUtilities.notScheduled', "Not Scheduled") : date;
+	public static convertToNextRun(date: string): string {
+		if (date.includes('1/1/0001')) {
```

Comment:
Not sure why you're converting this to an if - that's just adding extra lines of code for the exact same functionality.